### ISSUE TITLE
Enquiries on reproducing the results

### ISSUE THREAD

**Issue by lalithjets** [yuleiniu/cfvqa#13]
Hi,

First of all, great work and a good paper! 

I just want to clarify a few things! I followed the readme file and re-trained the following variants:
i) vqacp2 (smrl_baseline.yaml / smrl_cfvqa_sum.yaml /smrl_cfvqasimple_sum.yaml)
ii) vqa2 (smrl_baseline.yaml / smrl_cfvqa_sum.yaml /smrl_cfvqasimple_sum.yaml)
However, the evaluation results were different from the results reported in the paper.

1) Were the models originally trained for 22 epochs (as indicated in the YAML files)? if not, what's the recommended epoch to achieve results similar to the ones stated in the paper?
2) The paper reports the model's performance on the vqacp2 test set. However, the readme file states that there is no test set for vqacp2. Can I assume that the eval set and test set are the same for vqacp2?
3) Upon training, the model report results on logits_all, logits_vq, logits_cfvqa, logits_q and logits_v. How do I relate these results to the ones reported in the table?

Thank you for your time.

**Comment by yuleiniu** [comment]
Hi,
1. Yes, we train the models for 22 epochs following RUBi.
2. The val split in the codes refers to the test set of VQACP in the paper.
3. logits_cfvqa.json is for the results of CF-VQA. logits_vq.json is for the results of VQ-branch model. More explanations can be found [here](https://github.com/yuleiniu/cfvqa#train-a-model).

**Comment by lalithjets** [comment]
Thank you!
